src/EditorFeatures/Test2/IntelliSense/OldCompletion/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -0,0 +1,3914 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports System.Globalization
+Imports System.Threading
+Imports System.Threading.Tasks
+Imports Microsoft.CodeAnalysis.Completion
+Imports Microsoft.CodeAnalysis.CSharp
+Imports Microsoft.CodeAnalysis.Editor.CSharp.Formatting
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Extensions
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Utilities
+Imports Microsoft.CodeAnalysis.Options
+Imports Microsoft.CodeAnalysis.Tags
+Imports Microsoft.CodeAnalysis.Text
+Imports Microsoft.VisualStudio.Text
+Imports Microsoft.VisualStudio.Text.Editor
+Imports Microsoft.VisualStudio.Text.Editor.Commanding.Commands
+Imports Microsoft.VisualStudio.Text.Operations
+Imports Microsoft.VisualStudio.Text.Projection
+Imports Microsoft.VisualStudio.Utilities
+Imports Roslyn.Utilities
+
+Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense.OldCompletion
```

Comment:
❔ Why is this not registering as a file copy with only a namespace change?

Reply:
We do two changes at the same time:
1. Make a copy of the original file
2. Change the original file to use the new completion

I'm not sure if Git can handle both properly: display one as a copy and another as a change of the same file.

Is it possible?

---
In reply to: [208001346](https://github.com/dotnet/roslyn/pull/29016#discussion_r208001346) [](ancestors = 208001346)